README.rst

```diff
@@ -69,6 +69,20 @@ All required dependencies are installed automatically when you install ``autorej
 Optional dependencies are:
 
 * ``openneuro-py`` (>= 2021.10.1, for fetching data from `OpenNeuro.org <https://openneuro.org>`_)
+* ``torch`` (>= 2.0, for GPU acceleration)
+
+GPU Acceleration
+----------------
+
+For faster processing on large datasets, GPU acceleration is available via PyTorch::
+
+    pip install autoreject[gpu]
+
+To enable GPU acceleration, set the backend via environment variable::
+
+    export AUTOREJECT_BACKEND=torch
```

Comment:
would it be possible to set it within python with a context manager? Like MNE does for `use_3d_backend` or `set_3d_backend`